bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/NativeInterface.cs

```diff
@@ -1324,6 +1324,10 @@ protected override bool Free()
             internal static extern Status CryptoHash(
                 CandidateHandle handle,
                 out ElementModQ.ElementModQHandle crypto_hash);
+
+            [DllImport(DllName, EntryPoint = "eg_candidate_get_is_write_in",
+                CallingConvention = CallingConvention.Cdecl, SetLastError = true)]
+            internal static extern bool GetIsWriteIn(CandidateHandle handle);
```

Comment:
Not sure why this was missing from the candidate class.  It already existed in the C++.